### DISCUSSION TITLE
DJ Features integrated into API?

### DISCUSSION THREAD

**Discussion by nericollin** [tidal-music/discussions#114] [Ideas & Feature requests]
It's my understanding that the DJ subscription allows access to a master recording's stems so that they can be used for mixing and mashing and such.

Is there any way to implement this into the API to request a certain song and it's associated stems for a webapp?

**Comment by stefanmoro** [comment]
Hey @nericollin ,

DJ integrations are currently not available through TIDAL Developer. I suggest reaching out to [TIDAL Support](https://support.tidal.com) for more information on DJ integrations. Thanks
